torchaudio/functional/functional.py

```diff
@@ -994,6 +1000,47 @@ def spectral_centroid(
     return (freqs * specgram).sum(dim=freq_dim) / specgram.sum(dim=freq_dim)
 
 
+@_mod_utils.requires_module('torchaudio._torchaudio')
+def apply_codec(
+    waveform: Tensor,
+    sample_rate: int,
+    format: str,
+    compression: Optional[float] = None,
+    channels_first: bool = True,
+) -> Tensor:
+    r"""
+    Applies codecs as a form of augmentation
+
+    Args:
+        waveform (Tensor): Audio data. Must be 2 dimensional. See also ```channels_first```
+        sample_rate (int): Sample rate of the audio waveform
+        format (str): file format
+        channels_first (bool):
+            When True, both the input and output Tensor have dimension ``[channel, time]``.
+            Otherwise, they have dimension ``[time, channel]``.
+        compression (float):
+            Used for formats other than WAV. This corresponds to ``-C`` option of ``sox`` command.
+                * | ``MP3``: Either bitrate (in ``kbps``) with quality factor, such as ``128.2``, or
+                  | VBR encoding with quality factor such as ``-4.2``. Default: ``-4.5``.
+                * | ``FLAC``: compression level. Whole number from ``0`` to ``8``.
+                  | ``8`` is default and highest compression.
+                * | ``OGG/VORBIS``: number from ``-1`` to ``10``; ``-1`` is the highest compression
+                  | and lowest quality. Default: ``3``.
+            See the detail at http://sox.sourceforge.net/soxformat.html.
```

Comment:
I am making updates to the original docstring of compression parameter. Instead of copy-pasting, can you redirect? Something like ``See :py:func:`torchaudio.backend.sox_io_backend.save`.`` should do.

Reply:
Also, the order of arguments in docstring does not match with the actual order.